docs/integrations/builtin/credentials/hubspot.md

```diff
@@ -27,6 +27,10 @@ Create a [HubSpot](https://www.hubspot.com/) account.
 6. Use the provided **Client ID** and the **Client secret** with your HubSpot OAuth2 API credentials in n8n.
 7. If you are using the [HubSpot Trigger](/integrations/builtin/trigger-nodes/n8n-nodes-base.hubSpotTrigger/) node, copy the **App ID** along with the information in the previous step.
 8. Copy your OAuth Callback URL from the 'Create New Credentials' screen in n8n and paste in the **Redirect URL** section.
+
+!!! warning "Exact scope needed"
```

Comment:
```suggestion
!!! note "Exact scope needed"
```

Reply:
We try to save 'warning' for really major things: destructive actions, security topics etc. This addition is important, but not dangerous, so I think 'note' is better here.

Reply:
Ok, sounds good with that explanation.